<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestMoney</class>
 <widget class="QWidget" name="RequestMoney">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Request Payment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Request Payment</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelTargetUser">
       <property name="text">
        <string>Target User:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTargetUser"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAmount">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxAmount">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.01</double>
       </property>
       <property name="maximum">
        <double>1000000.00</double>
       </property>
       <property name="value">
        <double>0.01</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelMessage">
       <property name="text">
        <string>Message (Optional):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditMessage"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSendRequest">
     <property name="text">
      <string>Send Request</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonBackToHome">
     <property name="text">
      <string>Back to Home</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 